tdrs-backend/tdpservice/settings/common.py

```diff
@@ -368,6 +370,32 @@ class Common(Configuration):
 
     ENABLE_DEVELOPER_GROUP = True
 
+    # XMS CONFIG
+    '''
+    "issuer":"https://int.xms.hhs.gov/",
+    "authorization_endpoint":"https://int.xms.hhs.gov/ABaaS/oidc/authorize",
+    "token_endpoint":"https://int.xms.hhs.gov/ABaaS/oidc/token",
+    "userinfo_endpoint":"https://int.xms.hhs.gov/ABaaS/oidc/userinfo",
+    "jwks_uri":"https://int.xms.hhs.gov/ABaaS/oidc/certificates"
+    '''
+    XMS_ACR_VALUES = os.getenv(
+        'ACR_VALUES',
+        'http://idmanagement.gov/ns/assurance/ial/1'
+    )
+    XMS_AUTHORIZATION_ENDPOINT = 'https://int.xms.hhs.gov/ABaaS/oidc/authorize'
+
+    XMS_CLIENT_ASSERTION_TYPE = ''
+    XMS_CLIENT_ID = 'U2Xzpq6_ZA_5M~Z'
```

Comment:
should probably be an environment variable

Reply:
Correct, will do the change